### ISSUE TITLE
Compatible with 0.74.2?

### ISSUE THREAD

**Issue by skeith** [0x0049/Rocket.Chat.Dark#18]
I tried manual install, but it doesn't change the appearance

**Comment by DoM1niC** [comment]
Works 4 me ... Just the Modal Textcolor was broken, I compile the master from Branche without any Problems, #19 is the Pull Request with the Fix.

**Comment by eryg-kai** [comment]
Yup I haven't tested it on 0.74.2 myself but it should work. You're just pasting the CSS and hitting "save"?

**Comment by skeith** [comment]
> Yup I haven't tested it on 0.74.2 myself but it should work. You're just pasting the CSS and hitting "save"?

Yes. That's what I think how it works based on the readme. Maybe my understanding is incorrect?

**Comment by DoM1niC** [comment]
The last Dev is fully broken of RocketChat combinded with the current CSS

**Comment by Crazy-Hopper** [comment]
Hello, @0x0049 could you please clarify whether you have in your plans an update of this theme for version 1.0.0+ ?

**Comment by eryg-kai** [comment]
I gave it a quick test on the latest version and there are are definitely a lot of parts that need updating. I actually don't use Rocket.Chat anymore so at the moment I don't have any plans to make those updates, but if I find some time I might get around to it.

**Comment by Crazy-Hopper** [comment]
Well, I've been trying to get it back to the state where it was before and then some. Sharing my result here as a non-minified css. Be aware that it was quite a hacky approach and shouldn't be relied much upon. Some things were changed to my liking. YMMV as they say.
[RocketChat-DarkTheme-mangled-by-Crazy_Hopper.zip](https://github.com/0x0049/Rocket.Chat.Dark/files/3131285/RocketChat-DarkTheme-mangled-by-Crazy_Hopper.zip)
